VerbalExpressions.js

```diff
@@ -10,12 +10,20 @@
  *
  */
 
-// Define the collection class.
+/**
+* Define the VerbalExpression class.
+* @module VerbalExpressions
+*/
```

Comment:
Can you remove this declaration? It currently produces an empty "Modules"  link:
<img width="173" alt="screen shot 2016-01-27 at 09 14 06" src="https://cloud.githubusercontent.com/assets/2676795/12606469/674855c8-c4d6-11e5-8aee-ed6d097aeff1.png">